oxpy/CMakeLists.txt

```diff
@@ -4,12 +4,11 @@ SET(oxpy_SOURCES
 )
 
 IF(Python)
+	ADD_SUBDIRECTORY(pybind11)
 	ADD_DEFINITIONS(-DPYTHON_BINDINGS)
-	FIND_PACKAGE( PythonLibs 3 REQUIRED )
+	FIND_PACKAGE( Python 3 REQUIRED )
 	INCLUDE_DIRECTORIES( ${PYTHON_INCLUDE_DIRS} )
-	
-	ADD_SUBDIRECTORY(pybind11)
-	
+	MESSAGE(STATUS ${PYTHON_INCLUDE_DIRS} )
```

Comment:
I think we should either remove this line or print some message

Reply:
Is there a way of making this something that is shown only during debugging?

I think that it may be useful for confirming the location of the Python version that is being used so will add some more text to the message, but I don't think it is necessary for general use.

Reply:
There is no such facility as of now. I guess we might print it whenever the "Debug" or "G" cmake variables are set to ON.